src/libs/actions/Account.js

```diff
@@ -27,7 +27,7 @@ function fetchPriorityMode() {
         });
 }
 
-export default {
+export {
```

Comment:
the other action files didn't use exports I changed this to be consistent with the others.

Reply:
Would you mind undoing this? It seems a little unrelated to the task at hand.